extensions/default/src/ViewerLayout/ResizablePanelsHook.tsx

```diff
@@ -270,53 +281,55 @@ const useResizablePanels = (
       return;
     }
     const { width: panelGroupWidth } = resizablePanelGroupElemRef.current.getBoundingClientRect();
-    setRightPanelExpandedWidth((size / 100) * panelGroupWidth - sidePanelExpandedInsideBorderSize);
+    setRightPanelExpandedWidth(
+      (size / 100) * panelGroupWidth - panelGroupDefinition.shared.expandedInsideBorderSize
+    );
   }, []);
 
   return [
     {
       expandedWidth: leftPanelExpandedWidth,
-      collapsedWidth: sidePanelCollapsedWidth,
-      collapsedInsideBorderSize: sidePanelCollapsedInsideBorderSize,
-      collapsedOutsideBorderSize: sidePanelCollapsedOutsideBorderSize,
-      expandedInsideBorderSize: sidePanelExpandedInsideBorderSize,
+      collapsedWidth: panelGroupDefinition.shared.collapsedWidth,
+      collapsedInsideBorderSize: panelGroupDefinition.shared.collapsedInsideBorderSize,
+      collapsedOutsideBorderSize: panelGroupDefinition.shared.collapsedOutsideBorderSize,
+      expandedInsideBorderSize: panelGroupDefinition.shared.expandedInsideBorderSize,
       onClose: onLeftPanelClose,
       onOpen: onLeftPanelOpen,
     },
     {
       expandedWidth: rightPanelExpandedWidth,
-      collapsedWidth: sidePanelCollapsedWidth,
-      collapsedInsideBorderSize: sidePanelCollapsedInsideBorderSize,
-      collapsedOutsideBorderSize: sidePanelCollapsedOutsideBorderSize,
-      expandedInsideBorderSize: sidePanelExpandedInsideBorderSize,
+      collapsedWidth: panelGroupDefinition.shared.collapsedWidth,
+      collapsedInsideBorderSize: panelGroupDefinition.shared.collapsedInsideBorderSize,
+      collapsedOutsideBorderSize: panelGroupDefinition.shared.collapsedOutsideBorderSize,
+      expandedInsideBorderSize: panelGroupDefinition.shared.expandedInsideBorderSize,
       onClose: onRightPanelClose,
       onOpen: onRightPanelOpen,
     },
-    { direction: 'horizontal', id: viewerLayoutResizablePanelGroupId },
+    { direction: 'horizontal', id: panelGroupDefinition.groupId },
     {
-      defaultSize: resizablePanelDefaultSize,
-      minSize: resizablePanelDefaultSize,
+      defaultSize: leftResizablePanelMinimumSize,
```

Comment:
I'm not sure this is what we want here. Please try the following scenario...

- initially (on start up) have the left panel collapsed (i.e. closed can be set in the mode definition)
- expand the left panel and check which size is it and is it the one we want; I personally would expect it to be the larger of the two widths (i.e. the initial expanded width)

I can help you with this if you want.

Reply:
I see the problem, let me see if I can figure it out, thanks Joe

Reply:
Actually I can cover this in a subsequent PR if you would like. We will have a PR for the resize handle and I can take care of it there.

Reply:
I have something worked that i just pushed, if it doesn't work for you I can revert it and you can take this.